src/test/resources/dokumentasjon/features/hovedregler/lovvalg/13_bruker_er_doed_foer_input.feature

```diff
@@ -0,0 +1,38 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Regel 13: Er bruker død?
+
+  Scenariomal: Regel 13 - Bruker er død før input
+
+    Gitt følgende opplysninger om dødsfall i personhistorikken:
+      | Doedsdato       |
+      | <Dato>          |
```

Comment:
Bruk gjerne Dødsdato med ø som domenebegrep i Cucumber. Og <Dødsdato> istedet for <Dato>, for lesbarhetens skyld.

Reply:
done